manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -15,6 +15,8 @@ spec:
   replicas: 1
   template:
     metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
i was looking at the enhancement, and i did not see this annotation mentioned. is there anywhere i could read about it?

Reply:
Sorry, we had some additional renaming, you can find the relevant information under the PR - https://github.com/openshift/enhancements/pull/739

Reply:
awesome, thank you!